chia/data_layer/data_layer_util.py

```diff
@@ -43,6 +43,10 @@ def leaf_hash(key: bytes, value: bytes) -> bytes32:
     return Program.to((key, value)).get_tree_hash()  # type: ignore[no-any-return]
 
 
+def key_hash(key: bytes) -> bytes32:
+    return Program.to(key).get_tree_hash()  # type: ignore[no-any-return]
```

Comment:
I hesitate a bit to bring clvm into this hashing.  I see how it mirrors the `leaf_hash()` but IIRC that was chosen so that hashing of the left/right pairs up to the root could be done in clvm.  This hashing is purely a Python internal optimization and could be just a regular old hash.